template-processors/jinja/bin/processTemplates.sh

```diff
@@ -7,10 +7,10 @@ set -o errexit
 ## we assume that in $CLONED_PARAMETER_GIT_DIR there is a parameter file called parameters.yaml
 ## the result is a set of file stored in the $MANIFEST_DIR, with the same name as the templates, but with no extension. 
 
-for file in $CLONED_TEMPLATE_GIT_DIR/*.j2 ; do
+for file in ${CLONED_TEMPLATE_GIT_DIR}/*.j2 ; do
```

Comment:
Please put double quotes around the env var, e.g. like this:

    for file in "${CLONED_TEMPLATE_GIT_DIR}"/*.j2 ; do